client/keys/types.go

```diff
@@ -11,6 +11,17 @@ type AddNewKey struct {
 	Index    int    `json:"index,string,omitempty"`
 }
 
+// NewAddNewKey constructs a new AddNewKey request structure.
+func NewAddNewKey(name, password, mnemonic string, account, index int) AddNewKey {
```

Comment:
this name is weird, should probably rename it in another PR

Reply:
Well, it complies with constructors' idiomatic naming convention. We could rename the structure though.

Reply:
yeah it's weird but I think it's appropriate +1 to keep as Alessio's already done.